README.md

```diff
@@ -205,7 +205,7 @@ To use them, clone them into `$TESTGROUND_HOME/plans`. Same remark as above appl
 $ git clone https://github.com/libp2p/test-plans.git $HOME/testground/plans/libp2p
 $ git clone https://github.com/ipfs/test-plans.git $HOME/testground/plans/ipfs
```

Comment:
@coryschwartz we should replace these with the new `testground plan import` commands, no?

Reply:
Done in https://github.com/testground/testground/pull/938/commits/7db764bd2aadd8d232c1cc0160a0b88cbb096bb5.